pkg/operator/target_config_reconciler.go

```diff
@@ -1035,17 +1055,32 @@ func relieveAndMigrateProfile(profileCustomizations *deschedulerv1.ProfileCustom
 		setExcludedNamespacesForLowNodeUtilizationPlugin(profile.PluginConfigs[0].Args.Object.(*nodeutilization.LowNodeUtilizationArgs), includedNamespaces, excludedNamespaces, protectedNamespaces)
 	}
 
-	lowThreshold, highThreshold, err := getLowNodeUtilizationThresholds(profileCustomizations, false)
+	resourceNames := []v1.ResourceName{v1.ResourceCPU, v1.ResourceMemory, v1.ResourcePods}
+	args := profile.PluginConfigs[0].Args.Object.(*nodeutilization.LowNodeUtilizationArgs)
+
+	// profile defaults
+	const defaultActualUtilizationProfile = deschedulerv1.PrometheusCPUCombinedProfile
+	args.UseDeviationThresholds = true
+	query, err := utilizationProfileToPrometheusQuery(defaultActualUtilizationProfile)
 	if err != nil {
 		return nil, err
 	}
-
-	resourceNames := []v1.ResourceName{v1.ResourceCPU, v1.ResourceMemory, v1.ResourcePods}
-	args := profile.PluginConfigs[0].Args.Object.(*nodeutilization.LowNodeUtilizationArgs)
+	args.MetricsUtilization = &nodeutilization.MetricsUtilization{
+		Source: deschedulerapi.MetricsSource(v1alpha2.PrometheusMetrics),
+		Prometheus: &nodeutilization.Prometheus{
+			Query: query,
+		},
+	}
+	resourceNames = []v1.ResourceName{nodeutilization.MetricResource}
 
 	if profileCustomizations != nil {
 		// enable deviation
-		args.UseDeviationThresholds = profileCustomizations.DevDeviationThresholds != nil && *profileCustomizations.DevDeviationThresholds != ""
+		if profileCustomizations.DevDeviationThresholds != nil {
+			args.UseDeviationThresholds = *profileCustomizations.DevDeviationThresholds != ""
+		}
+		if profileCustomizations.DevLowNodeUtilizationThresholds != nil {
```

Comment:
The code needs to error when both `DevDeviationThresholds` and `DevLowNodeUtilizationThresholds` are non-nil:
```go
if profileCustomizations.DevDeviationThresholds != nil && profileCustomizations.DevLowNodeUtilizationThresholds != nil {
	return nil, fmt.Errorf("only one of DevLowNodeUtilizationThresholds and DevDeviationThresholds customizations can be configured simultaneously")
}
args.UseDeviationThresholds = profileCustomizations.DevDeviationThresholds != nil && *profileCustomizations.DevDeviationThresholds != ""
```